scope/src/doctor/runner.rs

```diff
@@ -362,6 +364,10 @@ fn prompt_user(prompt_text: &str, maybe_help_text: &Option<String>) -> bool {
     })
 }
 
+fn auto_approve(_prompt_text: &str, _maybe_help_text: &Option<String>) -> bool {
+    true
```

Comment:
I'm wondering if it is worth while to write the context of the prompt and help text with an additional log that it's being automatically applied.

I can't quite picture in my head what the user experience or backend traces look like.